packages/core/bootstrap/src/lib/metrics/index.ts

```diff
@@ -92,27 +110,19 @@ export const httpRequestsTotal = new client.Counter({
 export const httpRequestDurationSeconds = new client.Histogram({
   name: 'http_request_duration_seconds',
   help: 'A histogram bucket of the distribution of http request durations',
-  // we should tune these as we collect data, this is the default
-  // bucket distribution that prom comes with
-  buckets: [0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10],
-})
-
-export const cacheWarmerRequests = new client.Counter({
-  name: 'cache_warmer_requests',
-  help: 'The number of requests caused by the warmer',
-  labelNames: ['method', 'statusCode', 'apiKey', 'retry'] as const,
+  buckets: requestDurationBuckets,
 })
 
-export const httpRequestsCacheHits = new client.Counter({
-  name: 'http_requests_cache_hits',
-  help: 'The number of http requests that hit the cache',
-  labelNames: ['method', 'statusCode', 'apiKey', 'retry'] as const,
+export const dataProviderRequestAttempts = new client.Counter({
+  name: 'data_provider_request_attempts',
+  help: 'The number of http requests that attempt to request from a data provider',
+  labelNames: ['method', 'provider_status_code'] as const,
```

Comment:
This PR looks great, the one thing I'd change is turn this "attempts" metric into straight up "dataProviderRequests". I don't think it's using the provider status code either rn, I think just moving it after the req succeeds/fails would work well

Reply:
Disregard the second part, I see it is using it now